### PR TITLE
Explain how federation works for admins, various doc updates

### PR DESCRIPTION
**Pull request by Nutomic**

### CODE REVIEW

File: lemmy_utils/src/settings.rs [hunk 1]
```diff
@@ -134,7 +134,7 @@ impl Settings {
       .federation
       .allowed_instances
       .split(',')
-      .map(|d| d.to_string())
+      .map(|d| d.to_string().trim)
```
Comment (dessalines) [inline lemmy_utils/src/settings.rs:137]: This broke the build I think, I have this fixed in another branch, you can remove it.

File: docs/src/administration_federation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Federation 
+
+Note: ActivityPub federation is still under development. We recommend that you only enable it on test instances for now.
+
+To enable federation, change the setting `federation.enabled` to `true` and restart Lemmy.
```
Comment (dessalines) [inline docs/src/administration_federation.md:5]: I would say "in the `lemmy.hjson` file"

File: docs/src/administration_federation.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- `https://dev.lemmy.ml/u/nutomic` (User)
+- `https://dev.lemmy.ml/post/123` (Post)
+
+For an overview of how federation in Lemmy works on a technical level, check out our [Federation Overview](contributing_federation_overview.md).
+
+## Instance allowlist and blocklist
+
+The federation section of Lemmy's config has two variables `allowed_instances` and `blocked_instances`. These control which other instances Lemmy will federate with. Both settings take a comma separated list of domains, eg `dev.lemmy.ml, example.com`. You can either change those settings via `/admin`, or directly on the server filesystem. 
+
+It is important to note that these settings only affect sending and receiving of data between instances. If allow federation with a certain instance, and then remove it from the allowlist, this will not affect previously federated data. These communities, users, posts and comments will still be shown. They will just not be updated anymore. And even if an instance is blocked, it can still fetch and display public data from your instance.
```
Comment (dessalines) [inline docs/src/administration_federation.md:21]: Good adding this.

File: lemmy_utils/src/settings.rs (hunk 1 above)
Comment (Nutomic) [reply lemmy_utils/src/settings.rs]: Oops I didnt even mean to include this here. Fixed.

File: docs/src/administration_federation.md (hunk 2 above)
Comment (Nutomic) [reply docs/src/administration_federation.md]: done

**Pull request merged**